components/tokens/custom-spectrum/custom-large-vars.css

```diff
@@ -87,4 +87,12 @@ governing permissions and limitations under the License.
   --spectrum-treeview-item-indentation-large: 25px;
   --spectrum-treeview-item-indentation-extra-large: 30px;
   --spectrum-treeview-indicator-inset-block-start: 6px;
+
+  --spectrum-dial-border-radius: 20px;
+  --spectrum-dial-handle-position: 10px;
+  --spectrum-dial-handle-block-margin: 20px;
+  --spectrum-dial-handle-inline-margin: 20px;
+  --spectrum-dial-controls-margin: calc(20px / 2);
```

Comment:
the margin-inline-start `--spectrum-dial-controls-margin` is not correctly applying to the dial-control, I can see the token come through but it is not being applied. This is causing the dial to not be centered under the label. 

![Screenshot 2023-09-22 at 9 08 35 AM](https://github.com/adobe/spectrum-css/assets/63808889/8bcd0d2a-89c1-4a6c-8d31-dbb3253867fe)

Reply:
more on this here: https://github.com/adobe/spectrum-css/pull/1971#discussion_r1334533780